numba_rvsdg/core/datastructures/scfg.py

```diff
@@ -403,7 +423,7 @@ def find_exiting_and_exits(
                 exiting.add(inside)
         return sorted(exiting), sorted(exits)
 
-    def is_reachable_dfs(self, begin: str, end: str):  # -> TypeGuard:
+    def is_reachable_dfs(self, begin: str, end: str) -> bool:  # -> TypeGuard:
```

Comment:
I believe this comment was left over from previous attempts at `mypy` and can be removed:

```suggestion
    def is_reachable_dfs(self, begin: str, end: str) -> bool:
```

I removed it locally and ran:

```
 💣 zsh» mypy numba_rvsdg/core/datastructures/scfg.py
Success: no issues found in 1 source file
```

so that should probably be ok to remove.